content/en/docs/Reference/stage_modules.md

```diff
@@ -124,17 +124,19 @@ A list of files to write to disk.
 
 ```yaml
 #cloud-config
-
 stages:
   boot:
     - files:
-        - path: /tmp/bar
-      content: |
-        #!/bin/sh
-        echo "test"
-      permissions: 0777
-      owner: 1000
-      group: 100
+      - path: /tmp/bar
+        encoding: "b64" # "base64", "gz", "gzip", "gz+base64", "gzip+base64", "gz+b64", "gzip+b64"
```

Comment:
but the example below is not b64 is it?

Reply:
good point, I just copied it from the yip docs...

Reply:
done!